src/frequenz/sdk/timeseries/pv_pool/_pv_pool_reference_store.py

```diff
@@ -97,15 +97,20 @@ def __init__(  # pylint: disable=too-many-arguments
             name=f"System Bounds for PV inverters: {component_ids}",
             resend_latest=True,
         )
-        self.bounds_tracker: PVSystemBoundsTracker = PVSystemBoundsTracker(
-            self.component_ids,
-            self.status_receiver,
-            self.bounds_channel.new_sender(),
-        )
-        self.bounds_tracker.start()
+
+        self.bounds_tracker: PVSystemBoundsTracker | None = None
+        # In locations without PV inverters, the bounds tracker will not be started.
+        if self.component_ids:
+            self.bounds_tracker = PVSystemBoundsTracker(
+                self.component_ids,
+                self.status_receiver,
+                self.bounds_channel.new_sender(),
+            )
+            self.bounds_tracker.start()
 
     async def stop(self) -> None:
         """Stop all tasks and channels owned by the PVInverterPool."""
         await self.formula_pool.stop()
-        await self.bounds_tracker.stop()
+        if self.bounds_tracker is not None:
+            await self.bounds_tracker.stop()
```

Comment:
Not introduced by this PR, but just FYI, as I mentioned in other insteances and at least if this is a `BackgroundService`, stopping of sub-services should be done separately via `cancel()` and `wait()`, and leave `stop()` being just an alias for `self.cancel(); await self`.

Reply:
It is not a bug the way it is now, at least not in this layer, because this is in the `*reference_store`, which is supposed to be shared already.

But we will have to look at the outer layers, where we just removed the call to `*reference_store.stop()` as a temporary fix.